concepts/conditionals/introduction.md

```diff
@@ -83,9 +83,11 @@ end
 # => "1 is equal to 1"
 
 unless value < 2
-  "1 is not greater than 2"
+  "1 is not less than 2"
+else
+  "1 is less than 2"
 end
-# => "1 is greater than 2"
+# => "1 is less than 2"
```

Comment:
This may be the warranted change, in my opinion, as stated earlier.

Reply:
As Ellnix expressed, this is more than just a comment it is an expression of evaluation.  This will not end up existing in this form per the discussion on the forums, fortunately.